lib/pengine/pe_output.c

```diff
@@ -1898,6 +1898,32 @@ health2text(int health)
     }
 }
 
+/*!
+ * \internal
+ * \brief Convert a node type to a string representation
+ *
+ * \param[in] type  Node type
+ *
+ * \retval \c PCMK_VALUE_MEMBER if \p node_type is \c pcmk_node_variant_cluster
+ * \retval \c PCMK_VALUE_REMOTE if \p node_type is \c pcmk_node_variant_remote
+ * \retval \c PCMK__VALUE_PING if \p node_type is \c node_ping
+ * \retval \c "unknown" otherwise
+ */
+static const char *
+node_type2str(enum node_type type)
```

Comment:
Same commit as PCMK_XA_MAINTENANCE, just so that we can move node_type to the bottom of the variables when we add the final string constant for `node_xml()`.